lib/internal/bootstrap/pre_execution.js

```diff
@@ -20,6 +20,7 @@ function prepareMainThreadExecution() {
 
   initializeDeprecations();
   initializeESMLoader();
+  initializeFrozenIntrinsics();
```

Comment:
This should be moved before `initializeESMLoader`, which could execute user code specified with `--loader`

Reply:
Done, thanks.